src/librustdoc/html/render.rs

```diff
@@ -99,6 +99,9 @@ pub struct Context {
     /// publicly reused items to redirect to the right location.
     pub render_redirect_pages: bool,
     pub shared: Arc<SharedContext>,
+    /// The given user css file which allow to customize the generated
+    /// documentation theme.
+    pub css_file_extension: Option<PathBuf>,
```

Comment:
The issue comes from the fact that this option must be used in both `Context` and `SharedContext`. I tried to have it only in `SharedContext` but the code started to change way too much compared to the "gain".